src/applications/vaos/appointment-list/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Switch, Route } from 'react-router-dom';
+import { selectFeatureHomepageRefresh } from '../redux/selectors';
+import PageLayout from './components/AppointmentsPage/PageLayout';
+import AppointmentsPageV2 from './components/AppointmentsPage/AppointmentsPageV2';
+import AppointmentsPage from './components/AppointmentsPage/index';
+
+function AppointmentListSection({ featureHomepageRefresh }) {
```

Comment:
Following the pattern of the other sections, other detail page routes will go in here